### PR TITLE
Undefined api Key

### PR DESCRIPTION
**Pull request by raybooysen**
We test for `this._apiKey` and then append `self._apiKey` where self is undefined.

### CODE REVIEW

File: seq_logger.js [hunk 1]
```diff
@@ -216,7 +216,7 @@ class SeqLogger {
             };
             
             if (this._apiKey) {
-                opts.headers["X-Seq-ApiKey"] = self._apiKey;
```
Comment (raybooysen) [inline seq_logger.js]: This is causing no API key to go across the wire.

**Comment by nblumhardt** [comment]
Great find, thanks; looks like I was writing Smalltalk there...

**Comment by raybooysen** [comment]
No problem.  Is it possible to have this released so we can see this in bunyan-seq?

**Comment by nblumhardt** [comment]
Should be up now - 0.1.3 - let me know how you go! :-)

**Comment by raybooysen** [comment]
Perfect, thank you

**Pull request merged**